src/utils/GlobalContext.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
for fix this linter issue  - commit and push changes package.json file. even better commit and push all files with changes.
![Screenshot from 2023-12-07 18-11-36](https://github.com/mate-academy/react_todo-app-with-api/assets/33029864/5ec1f5aa-0d27-4c92-9299-7940a8a3b8b8)